src/backend/api/Fusion.Resources.Api/Controllers/Personnel/InternalPersonnelController.cs

```diff
@@ -255,6 +256,35 @@ public async Task<ActionResult> ResetAllocationState(string fullDepartmentString
 
             return NoContent();
         }
+
+        [HttpGet("/projects/{projectIdentifier}/resources/persons")]
+        public async Task<ActionResult> Search([FromRoute] PathProjectIdentifier projectIdentifier, [FromQuery(Name ="$search")] string search, [FromQuery] string? departmentFilter)
```

Comment:
Any reason for not using ODataQueryParams ?
Supports both $search & additional filters, and doesn't require to change the signature whenever you modify/add criterias.

Reply:
(I see same approach in departmentcontroller, but also multiple examples using OdataQueryParams)